src/primary-navigation-layout/terra-application-navigation/utils/propTypes.js

```diff
@@ -35,7 +35,7 @@ const userConfigPropType = PropTypes.shape({
   /**
    * User initials to be displayed within the avatar if no image is present.
    */
-  initials: PropTypes.string.isRequired,
+  initials: PropTypes.string,
```

Comment:
In the existing navigation component, we have logic to present the Generic avatar if initials are not provided. It doesn't seem like this needs to be required.